installer/pkg/validate/validate_test.go

```
@@ -377,40 +377,76 @@ func TestEmail(t *testing.T) {
 func TestCertificate(t *testing.T) {
 	const invalidMsg = "invalid certificate"
 	const privateKeyMsg = "invalid certificate (appears to be a private key)"
+	const badPem = "failed to parse certificate pem"
+	const certificate = `-----BEGIN CERTIFICATE-----
```

nit: for better maintainability, we could consider generating a throwaway certificate here instead of hardcoding a hand-generated one. not a blocker.